src/net/sourceforge/kolmafia/objectpool/ItemPool.java

```diff
@@ -3892,6 +3892,7 @@ public class ItemPool {
   public static final int STATS_ROM_CHIP = 11823;
   public static final int HASHING_VISE = 11826;
   public static final int GEOFENCING_RAPIER = 11827;
+  public static final int TOY_CUPID_BOW = 11836;
```

Comment:
> js Item.get("11836")

Returned: new-in-box toy Cupid bow
id => 11836

Reply:
Once this is fixed it's worth adding additional tests:
- one for successfully incrementing the pref when you don't get a drop
- one for resetting the pref when you swap familiars with the TCB equipped
- one for _not_ resetting the pref when you swap familiars and _don't_ have the TCB equipped
- one for setting the `_cupidBowFamiliars` pref when it's empty, rather than when it currently has a thing in it
(these should all be pretty straightforward; I'd recycle html for an existing test for ones where you don't want to get the drop. That being said I'm not a mafia dev so maybe this isn't necessary. but those are the cases I'd test, as just some guy)